molecule/command/init/role.py

```diff
@@ -106,7 +105,7 @@ def execute(self):
 @click.option(
     '--driver-name',
     '-d',
-    type=click.Choice(drivers()),
+    type=click.Choice(api.drivers()),
```

Comment:
You need to take care here and also use the solution in https://github.com/ansible/molecule/pull/2295 for completion?